src/routes/v1/search.route.ts

```diff
@@ -0,0 +1,17 @@
+import express, { Express } from "express";
+import SearchController from "../../controllers/search.controller";
+
+const searchRoute = (app: Express) => {
+	const accountController = new SearchController();
+	const router = express.Router();
+
+	// All paths have the prefix /api/v1/session/
+
+	router.get("/:searchString", (req, res) => {
+		accountController.getStocksSuggestions(req, res);
+	});
```

Comment:
```suggestion
	router.get("/:searchString", async (req, res) => {
		await accountController.getStocksSuggestions(req, res);
	});
```

Adding `async/await` here for two purposes:
1. Coding convention to show it is an asynchronous function
2. Avoid unexpected behavior